src/main/OPBuilder.ts

```diff
@@ -62,8 +80,13 @@ export default class OPBuilder {
     return this;
   }
 
+  //TODO: fill up with the actual values
   registrationBy(registrationBy: PassBy, refUri?: string): OPBuilder {
     this.responseRegistration = {
+      vpFormats: this.vpFormats,
+      issuer: undefined,
```

Comment:
This doesn't look right (the undefined and empty arrays. Needs to have a clear TODO

Reply:
fixed